code/game/gamemodes/game_mode.dm

```diff
@@ -533,4 +533,22 @@
 //By default nuke just ends the round
 /datum/game_mode/proc/OnNukeExplosion(off_station)
 	if(off_station < 2)
-		station_was_nuked = TRUE //Will end the round on next check.
\ No newline at end of file
+		station_was_nuked = TRUE //Will end the round on next check.
+
+/datum/game_mode/proc/crew_live_count(var/min_percent_as_num) //return number of survivors OR T/F if argument is set
+	var/players = 0
+	var/survivors = 0
+	for(var/i in GLOB.mob_list)

```

Comment:
use alive_mob_list or whatever @vuonojenmustaturska named it

Reply:
But i'm also counting the dead with min_percent_as_num